docs/source/model_design/concept_model/standard_results_formats/index.rst

```diff
@@ -0,0 +1,197 @@
+
+- Results format consistency enables generalizable results processing code (for researchers and engineers) and reduces the need to code exception cases
+
+- Shared language and classification of terms may aid in communication between researchers and engineers
+
+Formatting guide
+-----------------
+
+General considerations
+++++++++++++++++++++++
+
+1. Consistency with GBD names is preferred over internal/other types of consistency. Generally, GBD names (such as age_group_name, location_name, cause_name, etc.) should appear in Vivarium results as :code:`{gbd_name}.replace(" ","_").lower()`. Note that sometimes GBD names things in inconsistent ways. For instance, "6-11 months" and "12 to 23 months". However, this will allow us to easily merge Vivarium results and GBD data using their shared names.
+
+2. Successful execution of these standard formats depends on researcher documentation of requested model outputs within the simulation concept model document.
+
+.. todo::
+
+  Link to updated template doc when ready
+
+3. If a researcher-requested output does not clearly fit within the examples outlined below, researchers and engineers should meet to determine desired results formatting and document the format within the project concept model requested outputs section.
+
+Parameters in results dataframes
+++++++++++++++++++++++++++++++++
+
+.. list-table:: Parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Value type
+    - Definition
+    - Possible values
+  * - :code:`input_draw`
+    - integer
+    - Simulated input draw number
+    - [0, 999]
+  * - :code:`scenario`
+    - string
+    - Simulated scenario name
+    - * baseline
+      * alternative
+      * etc.
+  * - :code:`age`
+    - string
+    - Age group of stratum
+    - :code:`gbd_age_group_name.replace(" ","_").lower()`
+      
+      * early_neonatal
+      * 50_to_54
+      * etc.
+  * - :code:`sex`
+    - string
+    - Sex of stratum
+    - * male
+      * female
+  * - :code:`year`
+    - integer
+    - Simulated year
+    - * 2022 
+      * 2023
+      * etc.
+  * - :code:`{other_stratifying_variables}` (example: :code:`treatment`)
+    - string
+    - Some other variable by which to stratify values (in the same way as age, sex, year, etc.). Intervention coverage is a common example.
+    - Name and values of {other_stratifying_variables} to be defined in advance of implementation by researchers.
+
+
+      Example:
+
+      * covered
+      * uncovered
+  * - :code:`measure`
+    - string
+    - Description of what is being measured about the entity
+    - * ylds
+      * ylls
+      * deaths
+      * person_time
+      * transition_count
+      * first_moment
+      * second_moment
+  * - :code:`entity_type`
+    - string
+    - Type of entity being measured
+    - * cause
+      * rei
+      * {other}
+  * - :code:`entity`
+    - string
+    - Name of entity being measured
+    - :code:`gbd_{entity_type}_name.replace(" ","_").lower()`
```

Comment:
```suggestion
    - :code:`gbd_{entity_type}_name.replace(" ","_").lower()` if there is a corresponding GBD entity, otherwise agreed upon between research and engineering teams
```